docs/summary.md

```diff
@@ -0,0 +1,203 @@
+## Apollo Link
+
+Apollo Link is an extensible standard interface for modifying control flow of GraphQL queries and fetching GraphQL results.
+
+## Basic Usage
+
+Links can be used as a stand-alone client or with most major GraphQL clients.
+
+### Stand-alone
+
+`execute` accepts a standard GraphQL request with a query string or AST.
+`execute` returns an [Observable](https://github.com/zenparsing/zen-observable#api) that can be subscribed to.
+Links use observables to support GraphQL subscriptions and live queries, in addition to single response queries and mutations.
```

Comment:
Not just for subscriptions and live queries - also for polling! (meaning it's still valuable if you're just using HTTP)